src/lib/Fullscreen.js

```diff
@@ -2,20 +2,6 @@ import EventEmitter from 'events';
 import { CLASS_FULLSCREEN } from './constants';
 
 class Fullscreen extends EventEmitter {
-    /**
-     * [constructor]
-     *
-     * @return {Fullscreen} Fullscreen instance
-     */
-    constructor() {
-        super();
-
-        document.addEventListener('webkitfullscreenchange', this.fullscreenchangeHandler);
-        document.addEventListener('mozfullscreenchange', this.fullscreenchangeHandler);
-        document.addEventListener('MSFullscreenChange', this.fullscreenchangeHandler);
-        document.addEventListener('fullscreenchange', this.fullscreenchangeHandler);
```

Comment:
What're the reasons for removing these event listeners?

Reply:
We were playing around with those on Friday. @DanDeMicco has a PR addressing this soon :)

Reply:
Whoops I might have forgotten to properly rebase with this. It's unrelated to my commit. Fixing that now